resource/template/resource_test.go

```diff
@@ -56,12 +56,10 @@ func TestProcessTemplateResources(t *testing.T) {
 		t.Error(err.Error())
 	}
 
-	// Create the dest.
-	destFile, err := ioutil.TempFile("", "")
-	if err != nil {
-		t.Errorf("Failed to create destFile: %s", err.Error())
-	}
-	defer os.Remove(destFile.Name())
+	tempDestDir, err := ioutil.TempDir("", "")
```

Comment:
Why swallow the error here?  This seems like it should be marked as a failure like the original code.

Reply:
Yeah, that should definitely be propagated